common/app/views/fragments/items/elements/facia_cards/title.scala.html

```diff
@@ -5,17 +5,19 @@
 @import layout.FrontendLatestSnap
 @import implicits.ItemKickerImplicits._
 
-@icon = {
-@if(header.isExternal) {
-    @fragments.inlineSvg("external-link", "icon")
-}
+@headline() = {
 @if(header.quoted) {
-    @fragments.inlineSvg("garnett-quote", "icon")
+    <span class="@labelCssClasses fc-item__headline">
+        @fragments.inlineSvg("garnett-quote", "icon")
+        <span class="js-headline-text">@RemoveOuterParaHtml(header.headline)</span>
+    </span>
 }
+@if(header.isExternal) {
+    <span class="@labelCssClasses fc-item__headline">
+        <span class="js-headline-text">@RemoveOuterParaHtml(header.headline)</span>
+        @fragments.inlineSvg("external-link", "icon")
```

Comment:
it looks like we might need a default case here -- if we don't have a quote or an external link, where will the `header.headline` text get added?

Reply:
yep, good point!